gradle/base-information.gradle

```diff
@@ -7,8 +7,20 @@
 
 apply plugin: 'base'
 
+File versionFile = file( "${rootProject.projectDir}/gradle/version.properties" )
+
+if ( project.hasProperty( 'releaseVersion' ) ) {
+	if ( !project.hasProperty( 'developmentVersion' ) ) {
+		throw new GradleException(
+				"When releaseVersion parameter is specified also a developmentVersion have to be provided "
+		)
+	}
+	versionFile.text = "hibernateVersion=${project.property( 'releaseVersion' )}"
+}
+
 ext {
-	ormVersion = new HibernateVersion( '5.5.0-SNAPSHOT', project )
+	ormVersionFile = versionFile
+	ormVersion = HibernateVersion.fromFile( versionFile, project )
```

Comment:
Damn this will conflict massively with my JPA3 WIP :/

Maybe one of us should wait for the other to finish?